web/client/components/mapviews/settings/CompactRichTextEditor.jsx

```diff
@@ -58,7 +58,8 @@ function CompactRichTextEditor({
                     urlEnabled: true,
                     // disable the upload at the moment
                     // it will increase the size of the map too much
-                    uploadEnabled: false,
+                    // allows to set enable imge upload on/off via props
+                    uploadEnabled: props.uploadEnabled || false,
```

Comment:
Why are we allowing to upload images here? I think this is creating embedded base 64 images and I don't think it's a requirement of the task

Reply:
As per comments above, I guess image upload was disallowed for Geo-Stories only due to size issue (map + images), so decided to introduce a prop (false by default) if image upload needs to be allowed or just an embedded link